tutorials/introductory/usage.py

```diff
@@ -116,7 +116,7 @@
 # includes :class:`Text` objects, :class:`Line2D` objects,
 # :class:`collection` objects, :class:`Patch` objects ... (you get the
 # idea).  When the figure is rendered, all of the artists are drawn to
-# the **canvas**.  Most Artists are tied to an Axes; such an Artist
+# the **canvas**.  Most Artists are tied to an Axis; such an Artist
```

Comment:
"Axes" is correct here. What is meant is the `matplotlib.axes.Axes` object.

Reply:
I see, my mistake. Thanks